pkg/controller/plan/kubevirt.go

```diff
@@ -987,6 +994,9 @@ func (r *KubeVirt) dataVolumes(vm *plan.VMStatus, secret *core.Secret, configMap
 		annotations[AnnDefaultNetwork] = path.Join(
 			r.Plan.Spec.TransferNetwork.Namespace, r.Plan.Spec.TransferNetwork.Name)
 	}
+	if r.Plan.Spec.Warm && r.isOvirt() {
```

Comment:
possibly just `r.isOvirt()` is enough (only warm plan will enter now since the populator). but it's more self-explanatory

Reply:
so how would you cover cold migration to remote cluster?